app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/categories.controllers");
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
```

Comment:
You should have your error handling here - consider the 404 you have tested for and a 500 handler.